parachain/primitives/core/src/outbound.rs

```diff
@@ -367,3 +380,12 @@ pub enum AggregateMessageOrigin {
 }
 
 pub const ETHER_DECIMALS: u8 = 18;
+
+#[derive(Copy, Clone, Encode, Decode, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+pub struct TokenFees<Balance>
```

Comment:
I think this struct is a bit unnecessary. Its not really used much. We can just pass the two fees directly to the control pallet extrinsic.

Reply:
https://github.com/Snowfork/snowbridge/commit/f98004c058bf8841b1cc4d99583078e6358ae89c